Vue/src/components/GroupRowSelection/GroupRowComponent.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import DxLoadIndicator from 'devextreme-vue/load-indicator';
+import DxCheckBox, { type DxCheckBoxTypes } from 'devextreme-vue/check-box';
+import { onMounted, ref } from 'vue';
+import type { ColumnGroupCellTemplateData } from 'devextreme/ui/data_grid';
+
+const props = defineProps<{
+  groupCellData: ColumnGroupCellTemplateData,
+  initGroupRow: Function,
+}>();
+
+const iconSize = 18;
+
+let bindedLoading = ref(true);
+let bindedCheck = ref(false);
+let childKeys: any[] = [];
+
+onMounted(() => {
+  props.initGroupRow({
+    key: props.groupCellData.row.key,
+    setCheckedState: setCheckedState.bind(this)
```

Comment:
the following code will create a new func instance each time the initGroupRow method is called. If it's called for the same components it may be inefficient. 

Can we replace this logic with an arrow function? 
`setCheckedState: setCheckedState`

```
const setCheckedState = (...) => {

}
```

Reply:
Agree, this should be better. Fixed it.